docs/guidance/aws-eks-gpu-cluster.md

```diff
@@ -0,0 +1,58 @@
+# Prepare Amazon EKS cluster with GPU for KubeRay examples
+
+## Step 1: Create a Kubernetes cluster on Amazon EKS
+
+Follow the first two steps in this [AWS documentation](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#)
+to: (1) Create your Amazon EKS cluster (2) Configure your computer to communicate with your cluster.
```

Comment:
```suggestion
to: (1) create your Amazon EKS cluster and (2) configure your local machine to communicate with your cluster.
```

Reply:
These are the names of the sections in the following URL: https://docs.aws.amazon.com/eks/latest/userguide/getting-started-console.html#.

<img width="1209" alt="Screen Shot 2023-06-22 at 10 57 02 AM" src="https://github.com/ray-project/kuberay/assets/20109646/85d0d4fb-75e8-49ab-83f3-bba9a54761fa">

Reply:
Oh got it, let's keep it as-is then.